packages/strapi-plugin-graphql/services/schema-generator.js

```diff
@@ -104,7 +104,7 @@ const generateSchema = () => {
   }
 
   return {
-    typeDefs: gql(generatedSchema),
+    typeDefs: generatedSchema,
```

Comment:
Sorry I meant do return the generatedSchema directly has it contains the typeDefs. We need to move some of the logic that is applied afterwards in here to do so though.